__tests__/database/schema/ChannelListSchema.test.ts

```diff
@@ -226,22 +226,16 @@ describe('TESTING ChannelListSchema', () => {
         channel_type: 'channelType'
       });
 
-      ChannelListMemberSchema.getAll = jest.fn().mockResolvedValueOnce([]);
       UserSchema.fromDatabaseObject = jest.fn().mockResolvedValueOnce(null);
+      UserSchema.getAllByChannelId = jest.fn().mockResolvedValueOnce([]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The mock for `UserSchema.fromDatabaseObject` and `UserSchema.getAllByChannelId` methods are returning null and an empty array respectively. This might not be the best way to test these methods as it doesn't reflect a realistic scenario. Consider mocking them with actual data to ensure that the function behaves correctly when it receives valid input.
```diff
- UserSchema.fromDatabaseObject = jest.fn().mockResolvedValueOnce(null);
- UserSchema.getAllByChannelId = jest.fn().mockResolvedValueOnce([]);
+ UserSchema.fromDatabaseObject = jest.fn().mockResolvedValueOnce({/* some user object */});
+ UserSchema.getAllByChannelId = jest.fn().mockResolvedValueOnce([{/* some user object */}]);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->